Mage.Client/src/main/java/org/mage/card/arcane/CardRendererUtils.java

```diff
@@ -245,14 +245,12 @@ public static Color getCardTextColor(MageInt value, boolean drawAsDamaged, Color
         if (value != null) {
             int current = value.getValue();
             int origin = value.getModifiedBaseValue();
```

Comment:
Can you rename these two to something like `currentValue` and `baseValue`. More descriptive than "origin"

Reply:
Done